### DISCUSSION TITLE
Trying to run project on M2 Mac with Python3.10 interpretor and having issues with mysql-client. Getting the following error

### DISCUSSION THREAD

**Discussion by ridonkz** [PyMySQL/mysqlclient#711] [Q&A]
File "/Users/usera/Projects/venv/lib/python3.10/site-packages/MySQLdb/__init__.py", line 17, in <module>
    from . import _mysql
ImportError: dlopen(/Users/usera/Projects/venv/lib/python3.10/site-packages/MySQLdb/_mysql.cpython-310-darwin.so, 0x0002): symbol not found in flat namespace '_mysql_kill'

MySQL version - 8.3
Python - 3.10
Mac M2 
OS - Sonoma 14.4
mysql-client 8.3

**Comment by methane** [comment]
[Avoiding caching](https://pip.pypa.io/en/stable/topics/caching/#avoiding-caching)

**Reply by ridonkz** [reply]
did not work

**Reply by methane** [reply]
What is your complete step? What is output?
"Do not work" is not a good answer.
There is no respect to who are reading and responding to your question.
